.gitignore

```diff
@@ -1,5 +1,14 @@
-node_modules
-**/host_vars/iojs-*
-**/host_vars/node-*
-**/host_vars/test-*
-**/host_vars/release-*
+# tools/jenkins-status:
+node_modules/
+package-lock.json
+# setup secrets files, delete with setup/:
+setup/*/host_vars/iojs-*
+setup/*/host_vars/node-*
+setup/*/host_vars/test-*
+setup/*/host_vars/release-*
+
+# ansible/ secrets files (except templates).
+*.pyc
+ansible/host_vars/*
```

Comment:
Just verify that this is what was meant
```
ansible/host_vars/*
```
means ignore all files in `ansible/host_vars/` I'm not sure it applies recursively to subdirectories.
```
/ansible/host_vars/
```
is probably better (I think)

Reply:
I'm not sure there will ever be directories in `host_vars`, but if there are, maybe we'd want to keep them.

I'd rather leave it as is (this is what was there before) for now, and that way someone can edit the `.gitignore` if they ever want to add subfolders.